ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.hbs

```diff
@@ -84,8 +84,12 @@
     @title="No synced secrets yet"
     @message="Select secrets from existing KV version 2 engines and sync them to the destination."
   >
-    <LinkTo class="has-top-margin-xs" @route="secrets.destinations.destination.sync">
-      Sync secrets
-    </LinkTo>
+    <Hds::Link::Standalone
```

Comment:
I thought this styling + icon looked better:
![Screenshot 2023-12-21 at 2 56 17 PM](https://github.com/hashicorp/vault/assets/68122737/6c16a632-b0a5-42ea-b90d-d3833958708a)